Fix/snackbar (#174)
DHI/react-components: packages/react-components-lab/src/components/Snackbar/SnackbarProvider.stories.mdx

import { Meta, Story, Preview, Props } from '@storybook/addon-docs';

import * as stories from './SnackbarProvider.stories.tsx';
import SnackbarProvider from './SnackbarProvider';

<Meta title="Example/Snackbar" component={SnackbarProvider} />

# Snackbar

> `Snackbar` is used to display a message, known as notification / toast. To use this, you have to use `SnackbarProvider`
> component and `useSnackbar` hooks.

Reviewer: ```suggestion
`Snackbar` is used to display a message, known as notification / toast. For the component to work, you have to use the `SnackbarProvider` together with the `useSnackbar` hook.
```
